isis/src/control/objs/BundleUtilities/AbstractBundleObservation.cpp

```diff
@@ -345,6 +330,8 @@ QString AbstractBundleObservation::formatBundleOutputString(bool errorPropagatio
    */
   QString AbstractBundleObservation::bundleOutputCSV(bool errorPropagation) {
     // different in both
+    // TODO: either remove or update. 
+    return "Test";
```

Comment:
Do you think it's at all realistic to have something working for this at the parent class level, or should I just pull these out of this class?

Reply:
We can just make these functions pure virtual in the parent